python/interpret_community/common/model_wrapper.py

```diff
@@ -182,7 +182,7 @@ def wrap_model(model, examples, model_task):
         outputs a 1 dimensional array.
     :type model_task: str
     :return: The wrapper model.
-    :rtype model
+    :rtype: interpret_community.mimic.models.explainable_model.BaseExplainableModel.model
```

Comment:
This was missing closing ":". Then after adding that, the "model" type was ambiguous with the sphinx error show below. The error went away when I specify the exact model. If we are okay with ambiguous error in sphinx build then we can just leave as "model". Thoughs?


Error::

C:\Users\v-magelo\Documents\GitHub\interpret-community\python\interpret_community\common\model_wrapper.py:docstring of interpret_community.common.model_wrapper.wrap_model:None: WARNING: more than one target found for cross-reference 'model': interpret_community.mimic.models.explainable_model.BaseExplainableModel.model,
interpret_community.mimic.models.lightgbm_model.LGBMExplainableModel.model,
interpret_community.mimic.models.linear_model.LinearExplainableModel.model,
interpret_community.mimic.models.linear_model.SGDExplainableModel.model,
interpret_community.mimic.models.tree_model.DecisionTreeExplainableModel.model,
interpret_community.mimic.models.BaseExplainableModel.model,
interpret_community.mimic.models.LGBMExplainableModel.model,
interpret_community.mimic.models.SGDExplainableModel.model,
interpret_community.mimic.models.LinearExplainableModel.model,
interpret_community.mimic.models.DecisionTreeExplainableModel.model

Reply:
@imatiach-msft

Reply:
I think it should just be interpret_community.mimic.models.explainable_model.BaseExplainableModel, not interpret_community.mimic.models.explainable_model.BaseExplainableModel.model?

Reply:
oh wait, sorry, this is completely wrong. This is referring to any python model.  Just keep it as model if possible.  It's not any of the surrogate models.

Reply:
Went back to just model for now.